app/models/cloud_service.rb

```diff
@@ -0,0 +1,37 @@
+class CloudService < ApplicationRecord
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+  belongs_to :host
+  belongs_to :system_service
+  belongs_to :availability_zone
+
+  def fog_service
+    connection_options = {:service => source}
+    ext_management_system.with_provider_connection(connection_options) do |service|
+      service.services.find { |s| s.id.to_s == ems_ref }
```

Comment:
I updated that call with comment and created https://github.com/fog/fog-openstack/pull/88